### PR TITLE
better accessility for buttons

### LINKED ISSUE #2812: Accessibility in new Alarm Panel
HA Version: 0.88.0
Frontend Version: 20190220.0

Not a frontend developer here, the following is just an opinion. 
The latest ui update of the alarm panel is less accessible on the default theme than the previous one. 
The UI is cool and I enjoy it, but when you put it in the perspective of someone who has an hard time reading (eyes, ...), some problems may arise.

I came up with this solution:

Change the font of the numpad to something bigger and add an outline to the button.

What do you think?

### PR DESCRIPTION
**Pull request by mr-awk**
making the accessibility for the number buttons better.
Fixes #2812
image

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @mr-awk, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#3986).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 1]
```diff
@@ -294,10 +294,15 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
...
         margin-bottom: 5%;
+        margin-left: 1%;
+        margin-right: 1%;
         width: 30%;
         padding: calc(var(--base-unit));
         font-size: calc(var(--base-unit) * 1.1);
         box-sizing: border-box;
+        border-style: solid;
+        border-color: #03a9f4;
+        border-radius: 4px;
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts]: For the borders, use `outlined` on the `mwc-button`:
```
<mwc-button outlined  dense>border button</mwc-button>
```

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 2]
```diff
@@ -294,10 +294,15 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
 
       #keypad mwc-button {
         margin-bottom: 5%;
+        margin-left: 1%;
+        margin-right: 1%;
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts]: Remove all the margin and add less padding:
```
        padding: calc(var(--base-unit) / 3);
```

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-alarm-panel-card.ts]: I think we should drop that `--base-unit` var, as it doesn't do much? And I see there is a hard `300px` width on the `div#keypad`, that should go, should be `100%`

Comment (mr-awk) [reply src/panels/lovelace/cards/hui-alarm-panel-card.ts]: @bramkragten I will take a look at it

**Pull request closed without merging**